server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java

```diff
@@ -451,10 +451,10 @@ public void testSelectFromTableConstraints() throws Exception {
         assertThat(response.rows()[3][1]).isEqualTo(sqlExecutor.getCurrentSchema() + "_test_col3_not_null");
         assertThat(response.rows()[3][2]).isEqualTo("test");
         assertThat(response.rows()[4][0]).isEqualTo("CHECK");
-        assertThat(response.rows()[4][1]).isEqualTo("unnecessary_check");
+        assertThat(response.rows()[4][1]).isEqualTo("chk_1");
```

Comment:
Why does this change?

Reply:
Because previously we first passed mapping as string
1. https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/execution/ddl/tables/TransportCreateTableAction.java#L196 We created `CreateIndexRequest` out of `CreateTableRequest` to be able to direcly use `TransportCreateIndexAction.masterOperation`
2.
and then service created map out of this string 
https://github.com/crate/crate/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java#L325
which I think loses order as it uses parser.map(). For preserving there is `mapOrdered`

Current order is actually corresponds to the declaration order, I think it makes more sense?